### PR TITLE
Improved Regex

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Updated regex-lookup.yml by removing redundant characters in the regular expressions.

### CODE REVIEW

**Review by generalhenry — APPROVED** [review]

File: tests/cli/lookup/regex-lookup.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 logs-all: ([a-zA-Z0-9\.\-:\"\s\_\t\n$&?/\[\]%><,;()+='*{}\\]*)
 logs-metadata: (timestamp:"[A-Z0-9\.\-:\"]+\s+time_id:"[A-Z0-9\.\-:\"]+\s+service_id:"[a-z0-9$\"]+\s+service_name:"[a-z0-9\-$\"]+\s+message:"[a-zA-Z0-9\./\-:\s\_\"=\t\n\$\\&%?\[\]\%\>\<\,\;\+$\"]+\s+container_id:"[a-z0-9$\"]+\s+node_id:"[a-z0-9$\"]+\s+task_id:"[a-z0-9$\"]+\s+task_name:"[a-z0-9\.\-$\"]+)
 logs-numbered: ([a-zA-Z0-9\.\-:\"\s\_\t\n\$&%?/\[\]\%\>\<\,\;]*){10}
-docker-service-list: ID\s+NAME\s+REPLICAS\s+IMAGE\s+COMMAND\s*\n([a-zA-Z0-9\s\.\-:/]*){1,}
+docker-service-list: ID\s+NAME\s+REPLICAS\s+IMAGE\s+COMMAND\s*([a-zA-Z0-9\s\.\-:/]*){1,}
```
Comment (subfuzion) [inline tests/cli/lookup/regex-lookup.yml]: Can you verify this is correct ("COMMAND")?

**Review by subfuzion — CHANGES_REQUESTED** [review]

**Comment by subfuzion** [comment]
Please resolve conflicts and double-check the regex for docker-service-list so I can merge. Thanks.

**Comment by neha-viswanathan** [comment]
`docker-service-list` is not being used. So I've removed it.

**Review by subfuzion — APPROVED** [review]

**Comment by subfuzion** [comment]
LGTM

**Pull request merged**